daft/io/_csv.py

```diff
@@ -69,11 +62,6 @@ def read_csv(
     if isinstance(path, list) and len(path) == 0:
         raise ValueError("Cannot read DataFrame from from empty list of CSV filepaths")
 
-    if schema_hints is not None:
-        warnings.warn("schema_hints is deprecated and will be removed in a future release. Please use schema instead.")
```

Comment:
thoughts on raising an error instead rather than deleting the kwarg? Otherwise it may not be actionable to the user.